### ISSUE TITLE
Request for tooltips in the user interface

### ISSUE THREAD

**Issue by andrew-morrison** [bodleian/medieval-mss#101]
## Detailed Description
@holfordm has suggested tooltips (or whatever you call popup information boxes) to provide help to users and make them aware of things:

> The need for tooltips (or whatever they are called) was noted at the Fihrist meeting - the Fihrist website currenlty has them in the collection statistics box http://www.fihrist.org.uk/ but they are lacking from the Medieval site currently

## Context
For example, when subtleties of the data might need to be communicated to end users. Such as why the numbers against all the options in filters/facets don't add up to the total number of matched records. See discussion in #100.

## Link to Page
The filters boxes on the left [here](https://medieval.bodleian.ox.ac.uk/?f%5Btype%5D%5B%5D=person) could have question marks like the boxes on the left [here](http://www.fihrist.org.uk/browse/people).

## Possible Implementation
This is all user interface work, and probably quite substantial to store the text somewhere, associate it with the relevant user interface elements, and develop and test an implementation which meets accessibility requirements and works in both desktop and tablet browsers.

**Comment by ahankinson** [comment]
Assigning to Gabriel for comment

**Comment by IgnatG** [comment]
I'm not sure if a tooltip is the right way to do it since they are normally meant for short(3-4 words) descriptions. 
Nevertheless, if it's something that we need to add, then I can look into it. -- Maybe come up with another option. 
The problem is that this will take longer to implement and at the moment I can start working on it in July (the earliest) next year. If it needs to be implemented sooner, then you will have to talk to Judith about it.

**Comment by ahankinson** [comment]
Perhaps a note could be added to the "Help" page about the contents of the facets?

**Comment by IgnatG** [comment]
Yes. This is something that can be implemented straight away - @holfordm has to do it though. 
I'm not entirely sure if people are actually looking at the "Help" page that often to notice that we've added something new there but I think it can be a start.

**Comment by holfordm** [comment]
This is an interim solution although I strongly suspect that most people don't look at the help pages. 
Possible danger of the help page becoming crowded and wieldy? @ahankinson to what extent could more detailed help/about information go on a wiki in this repo (with links to/from main site of course)?

**Comment by ahankinson** [comment]
If we're worried about the Help pages not getting read by users, I would suspect that a wiki page here would be read by even fewer, even with links to/from.

I would reserve the wiki pages here for documentation about the service itself, or for information about how to catalogue the data and keep anything related to users on the site itself.

**Comment by andrew-morrison** [comment]
Maybe an FAQs section in the help page? Such as...

> Q: Why don't the numbers in the "Limit results by" filters add up to the total number of records?
> A: Some records belong to many categories (e.g. multiple languages used in a single manuscript) and some don't have as much information as others (e.g. only those places that are known to have been involved in the creation of manuscripts in this catalogue will have a role).

...and...

> Q: How do I exclude words from a search?
> A: Use a minus sign (e.g. astro* -astrology which finds every starting with "astro", like "astronomy", but excluding "astrology".)
